nav2_rviz_plugins/src/nav2_panel.cpp

```diff
@@ -777,7 +778,12 @@ void Nav2Panel::handleGoalSaver()
 void
 Nav2Panel::onInitialize()
 {
-  auto node = getDisplayContext()->getRosNodeAbstraction().lock()->get_raw_node();
+  node_ptr_ = getDisplayContext()->getRosNodeAbstraction().lock();
+  if (node_ptr_ == nullptr) {
+    // The node no longer exists, so just don't initialize
```

Comment:
For all: Some kind of log, even if `cout`, that this is the case would be nice (for all instances)

Reply:
Good call.  Fixed in 864a450.